skorch/callbacks/logging.py

```diff
@@ -747,32 +756,30 @@ class SacredLogger(Callback):
 
     To use this logger, you first have to install Sacred:
 
-    $ python -m pip install sacred
+    ``$ python -m pip install sacred``
```

Comment:
I think this works for Sphinx?

```suggestion
    .. code-block::
      $ python -m pip install sacred
```

Reply:
I used code blocks for the 2 shell commands in this docstring.